spec/views/hq/orphans/_form_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/orphans/_form.html.erb", type: :view do
+  let(:orphan_statuses) { OrphanStatus.all }
+  let(:orphan_sponsorship_statuses) { OrphanSponsorshipStatus.all }
+  let(:provinces) { Province.all }
+  let(:orphan) { build_stubbed :orphan }
+
+  before :each do
+    assign(:orphan_statuses, orphan_statuses)
+    assign(:orphan_sponsorship_statuses, orphan_sponsorship_statuses)
+    assign(:provinces, provinces)
+    assign(:orphan, orphan)
+  end
+
+  specify 'has a form' do
+    render
+
+    assert_select 'form'
+  end
+
+  describe '"Cancel" button' do
+    specify 'when no :id' do
```

Comment:
Please update these tests to reflect the fact that the form is not used for creating new orphan records.